Bigtable/owlbot.py

```diff
@@ -14,50 +14,45 @@
 
 """This script is used to synthesize generated parts of this library."""
 
+import logging
+from pathlib import Path
 import subprocess
+
 import synthtool as s
-import synthtool.gcp as gcp
-import logging
+from synthtool.languages import php
+from synthtool import _tracked_paths
 
 logging.basicConfig(level=logging.DEBUG)
 
-gapic = gcp.GAPICBazel()
-common = gcp.CommonTemplates()
-
-library = gapic.php_library(
-    service='bigtable',
-    version='v2',
-    bazel_target=f'//google/bigtable/v2:google-cloud-bigtable-v2-php',
-)
-
-# copy all src except handwritten partial veneers
-s.move(library / f'src/V2/Gapic')
-s.move(library / f'src/V2/resources')
+src = Path(f"../{php.STAGING_DIR}/Bigtable").resolve()
+dest = Path().resolve()
 
-# copy proto files to src also
-s.move(library / f'proto/src/Google/Cloud/Bigtable', f'src/')
-s.move(library / f'tests/')
+# Added so that we can pass copy_excludes in the owlbot_main() call
+_tracked_paths.add(src)
 
-# copy GPBMetadata file to metadata
-s.move(library / f'proto/src/GPBMetadata/Google/Bigtable', f'metadata/')
-
-# Bigtable Admin also lives here
-admin_library = gapic.php_library(
-    service='bigtable-admin',
-    version='v2',
-    bazel_target='//google/bigtable/admin/v2:google-cloud-bigtable-admin-v2-php'
+# Excluding gapic_metadata.json and the partial veneer files.
+php.owlbot_main(
+    src=src,
+    dest=dest,
+    copy_excludes=[
+        src / "*/src/*/gapic_metadata.json",
+        src / "*/src/*/*.php"
```

Comment:
As in the other one, can we be more specific and use `"*/src/*/*Client.php"` here? Also, can we add a comment for why we are excluding `gapic_metadata.json`?

Reply:
It's just preserving the behavior. We can copy the json file.

Reply:
Done